### DISCUSSION TITLE
Avoid flatbuffers duplicate class error

### DISCUSSION THREAD

**Discussion by LorenzoMorelli** [sceneview/sceneform-android#45] [Ideas]
Hi, first of all thanks you to maintain this project!!
I would like suggest you to consider (if possible) to export flatbuffers dependency from project package in order to avoid problems with importing of other library (in my case I had some problems with tensorflow-lite and I resolved importing sceneform as module without using gradle).

Nice coding!!

**Comment by ThomasGorisse** [comment]
Hi,
You're right, this is something I was thinking about. But I don't have time for this right now.
Can you create a PR ?

**Reply by LorenzoMorelli** [reply]
What Does "PR" stand for?

**Reply by Flushuine** [reply]
Any updates regarding this issue?

**Comment by ThomasGorisse** [comment]
PR = Pull Request
It's quite easy to do directly from Android Studio or GitHub.
